recipes/ffmpeg/all/conanfile.py

```diff
@@ -0,0 +1,352 @@
+               "mp3lame": [True, False],
+               "fdk_aac": [True, False],
+               "webp": [True, False],
+               "ssl": [False, "openssl", "securetransport"],
+               "alsa": [True, False],
+               "pulse": [True, False],
+               "vaapi": [True, False],
+               "vdpau": [True, False],
+               "xcb": [True, False],
+               "appkit": [True, False],
+               "avfoundation": [True, False],
+               "coreimage": [True, False],
+               "audiotoolbox": [True, False],
+               "videotoolbox": [True, False],
+               "qsv": [True, False]}
+    default_options = {"shared": False,
+                       "fPIC": True,
+                       "postproc": True,
+                       "zlib": True,
+                       "bzlib": True,
+                       "lzma": True,
+                       "iconv": True,
+                       "freetype": True,
+                       "openjpeg": True,
+                       "openh264": True,
+                       "opus": True,
+                       "vorbis": True,
+                       "zmq": False,
+                       "sdl2": False,
+                       "x264": True,
+                       "x265": True,
+                       "vpx": True,
+                       "mp3lame": True,
+                       "fdk_aac": True,
+                       "webp": True,
+                       "ssl": "openssl",
+                       "alsa": True,
+                       "pulse": True,
+                       "vaapi": True,
+                       "vdpau": True,
+                       "xcb": True,
+                       "appkit": True,
+                       "avfoundation": True,
+                       "coreimage": True,
+                       "audiotoolbox": True,
+                       "videotoolbox": True,
+                       "qsv": True}
+    generators = "pkg_config"
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _is_mingw_windows(self):
+        return self.settings.os == "Windows" and self.settings.compiler == "gcc" and os.name == "nt"
+
+    @property
+    def _is_msvc(self):
+        return self.settings.compiler == "Visual Studio"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+            destination=self._source_subfolder, strip_root=True)
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def validate(self):
+        if self.settings.os == "Windows":
+            if self.options.qsv:
+                raise ConanInvalidConfiguration("intel_media_sdk not available yet in CCI")
+        if self.settings.os in ["Linux", "FreeBSD"]:
+            raise ConanInvalidConfiguration("Linux not supported yet")
```

Comment:
You say `sdl` is missing.
Is that an essential lib? Isn't there an `--enable-sdl` option?

What system libraries are missing on Linux?

Reply:
I checked the bincrafters formula: https://github.com/bincrafters/community/blob/main/recipes/ffmpeg/all/conanfile.py#L183-L194

The system depencies seems to be available in CCI so Linux should work.